core/src/main/resources/db/migration.sql

```diff
@@ -15,3 +15,10 @@ ALTER TABLE cna_event AUTO_INCREMENT=1;
 UPDATE info SET DB_SCHEMA_VERSION="1.2.0";
 ALTER TABLE mutation add UNIQUE KEY `UQ_MUTATION_EVENT_ID_GENETIC_PROFILE_ID_SAMPLE_ID` (`MUTATION_EVENT_ID`,`GENETIC_PROFILE_ID`,`SAMPLE_ID`);
 ALTER TABLE sample_profile add UNIQUE KEY `UQ_SAMPLE_ID_GENETIC_PROFILE_ID` (`SAMPLE_ID`,`GENETIC_PROFILE_ID`);
+
+##version: 1.2.1
+SET @s = (SELECT IF(    (SELECT COUNT(*) FROM INFORMATION_SCHEMA.COLUMNS WHERE table_name = 'cancer_study' AND table_schema = DATABASE() AND column_name = 'STATUS') > 0,  "SELECT 1", " ALTER TABLE cancer_study ADD STATUS int(1) DEFAULT NULL"));
```

Comment:
Pieter, what is this code for? Could you add some comments?

Since we haven't release 1.2.0, I think we should squeeze it into 1.2.0? Since the code is already merge. I'll do a quick fix now.